sandbox/settings.py

```diff
@@ -240,11 +243,19 @@ class Base(StyleguideMixin, DRFMixin, RichieCoursesConfigurationMixin, Configura
             "SELECTOR_REGEX": values.Value(
                 r".*", environ_name="EDX_SELECTOR_REGEX", environ_prefix=None
             ),
+            "JS_SELECTOR_REGEX": values.Value(
+                r".*", environ_name="EDX_SELECTOR_JS_REGEX", environ_prefix=None
+            ),
             "COURSE_REGEX": values.Value(
                 r"^(?P<course_id>.*)$",
                 environ_name="EDX_COURSE_REGEX",
                 environ_prefix=None,
             ),
+            "JS_COURSE_REGEX": values.Value(
```

Comment:
ditto. The LMS_BACKENDS setting will be removed, so I think you should use your own settings.